openstack_plugin/utils.py

```diff
@@ -472,19 +473,154 @@ def update_runtime_properties_for_operation_task(operation_name,
         unset_runtime_properties_from_instance(ctx_node_instance)
 
 
-def handle_external_resource(ctx_node_instance,
-                             openstack_resource,
-                             existing_resource_handler=None,
-                             **kwargs):
+def lookup_remote_resource(_ctx, openstack_resource):
+    """
+    This method will try to lookup openstack remote resource based on the
+    instance type
```

Comment:
What is a "remote" resource"?

Reply:
It is the resource on the openstack side, so basically it will fetch the data we need from the openstack API